virttest/virt_vm.py

```diff
@@ -1049,66 +1049,69 @@ def wait_for_login(self, nic_index=0, timeout=LOGIN_WAIT_TIMEOUT,
                 when remote login (ssh, rss) failed.
         :return: A ShellSession object.
         """
-        def print_guest_network_info():
+        def print_guest_network_info(session=None):
```

Comment:
session parameter is not used inside this function. so maybe removing it is ok

Reply:
OK, removed.